db/src/db.rs

```diff
@@ -518,10 +523,7 @@ impl DB {
                value_type_tag0 SMALLINT NOT NULL,
```

Comment:
These tables definitely deserve a comment about the flag compression.  And flags should still be a `TINYINT`, although this doesn't make much difference in SQLite, IIUC.